src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -79,7 +79,7 @@ function getPaymentMethodTitle(
             },
             [PaymentMethodId.Bolt]: {
                 logoUrl: '',
-                titleText: language.translate('payment.credit_card_text'),
+                titleText: (method.config.displayName?.toLowerCase() === 'credit card' ? language.translate('payment.credit_card_text') : method.config.displayName) || '',
```

Comment:
For here why don't we write a method, something like `getBoltDisplayName(config) || ''`?

Reply:
I thought to write a common method for this case, because we have almost the same for Klarna and some other payment providers.

Reply:
I will create the method and let you know. Thanks! 🙏